deploy/tutorials/tutorial-firebase.md

```diff
@@ -364,23 +370,7 @@ new lines are not required):
 }
 ```
 
-## Deploy the application
-
-Now let's deploy the application:
-
-1. Go to https://gist.github.com/new and create a new gist, ensuring the
-   filename of the gist ends with `.js`.
-
-   > For convenience the whole application is hosted at
-   > https://deno.com/examples/firebase.js. You can skip creating a gist if you
-   > want to try the example without any modification, or click the link at the
-   > bottom of the tutorial.
-
-2. Copy the _Raw_ link of the saved gist.
-3. In your project on `dash.deno.com`, click the **Deploy URL** button and enter
-   the link to the raw gist in the URL field.
-4. Click the **Deploy** button and copy one of the URLs displayed in the
-   **Domains** section of the project panel.
+Click to save the variables.
```

Comment:
nit: it would be nicer to clarify what to click maybe?

```suggestion
Click **Save** button to save the variables.
```

Reply:
The reason I didn't do this was cos the text in the button changes depending on how many variables you have.